tests/globalhelper/helper.go

```diff
@@ -264,3 +268,134 @@ func AppendContainersToDeployment(deployment *v1.Deployment, containersNum int,
 
 	return deployment
 }
+
+// CreateAndWaitUntilReplicaSetIsReady creates replicaSet and wait until all replicas are up and running.
+func CreateAndWaitUntilStatefulSetIsReady(statefulSet *v1.StatefulSet, timeout time.Duration) error {
+	runningReplica, err := APIClient.StatefulSets(statefulSet.Namespace).Create(
+		context.Background(),
+		statefulSet,
+		metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+
+	Eventually(func() bool {
+		status, err := isStatefulSetReady(runningReplica.Namespace, runningReplica.Name)
+		if err != nil {
+			glog.V(5).Info(fmt.Sprintf(
+				"statefulSet %s is not ready, retry in 5 seconds", runningReplica.Name))
```

Comment:
retry in %d seconds -> use a var/const for this "5" number that you're using in the Eventually functions so in case you need to tweak/tune it, this message is also updated.